pyanaconda/network.py

```diff
@@ -30,41 +30,55 @@
 import time
 import threading
 import re
-import dbus
 import ipaddress
 
-from pyanaconda.simpleconfig import SimpleConfigFile
-
-from pyanaconda import nm
 from pyanaconda.flags import flags
 from pyanaconda.core.i18n import _
 from pyanaconda.core.regexes import HOSTNAME_PATTERN_WITHOUT_ANCHORS
 from pyanaconda.core.configuration.anaconda import conf
 from pyanaconda.modules.common.constants.services import NETWORK, TIMEZONE
 from pyanaconda.modules.common.task import sync_run_task
 from pyanaconda.payload.livepayload import LiveImagePayload
+from pyanaconda.dbus.structure import apply_structure
+from pyanaconda.modules.common.structures.network import NetworkDeviceInfo
 
 from pyanaconda.anaconda_loggers import get_module_logger
 log = get_module_logger(__name__)
 
-sysconfigDir = "/etc/sysconfig"
-netscriptsDir = "%s/network-scripts" % (sysconfigDir)
 DEFAULT_HOSTNAME = "localhost.localdomain"
 
 network_connected = None
 network_connected_condition = threading.Condition()
 
+_nm_client = None
```

Comment:
I wonder whether we really need a global proxy of the Network Manager. Maybe the function `get_nm_client` could always return a new instance. The spokes keep their instances anyway and otherwise it doesn't look like we heavily call the function.

Reply:
I'll think about it, I don't like having the private global very much. But I'll leave this to another PR.